crates/typos-dict/tests/verify.rs

```diff
@@ -19,26 +19,21 @@ fn verify() {
 }
 
 fn generate<W: std::io::Write>(file: &mut W, dict: &[u8]) {
-    let mut rows = Dict::new();
-    csv::ReaderBuilder::new()
+    let mut reader = csv::ReaderBuilder::new()
         .has_headers(false)
         .flexible(true)
-        .from_reader(dict)
+        .from_reader(dict);
+    let rows: Dict = reader
         .records()
         .map(Result::unwrap)
-        .for_each(|r| {
+        .map(|r| {
             let mut i = r.iter();
             let mut typo = i.next().expect("typo").to_owned();
             typo.make_ascii_lowercase();
             let typo = UniCase::new(typo);
-            rows.entry(typo)
-                .or_insert_with(BTreeSet::new)
-                .extend(i.map(|c| {
-                    let mut c = c.to_owned();
-                    c.make_ascii_lowercase();
-                    c
-                }));
```

Comment:
This change is incorrect.

This will cause the last implementation of a typo to "win" when the intention is the merge the entries

Reply:
Ah good catch, fixed.